apps/explorer/src/app/components/links/block-link/block-link.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Routes } from '../../../routes/route-names';
+import { Link } from 'react-router-dom';
+
+export type BlockLinkProps = {
```

Comment:
suggestion (?): same for the BlockLinkProps, would be nicer to merge with the Link component's props type to ensure you can only pass in valid things

Reply:
👍 As above - this worked nicely. Applied it across all the new Link components